### PR TITLE
before_create: article preview; pagination; other defaults

### PR DESCRIPTION
**Pull request by jkao1**
- Generate article previews on article creation.
- Pagination works! Removed limit arguments on several GraphQL resolvers
- Added `false` default to Article.is_published

### CODE REVIEW

File: app/models/article.rb [hunk 1]
```diff
@@ -15,21 +18,32 @@ class Article < ApplicationRecord
...
     Article
-      .joins("LEFT JOIN sections ON articles.section_id = sections.id")
+      .joins(:section)
       .order("articles.rank + 3 * sections.rank + 12 * articles.issue + 192 * articles.volume DESC")
   end
 
-   # TODO: generate cleaned/truncated content for summary if no summary provided
+  private
+    def init
+      if self.summary.nil? || self.summary.empty?
```
Comment (NicholasLYang) [inline app/models/article.rb]: Looks good. The only issue I see is the html tags being counted as words, but that's a little acceptable

Comment (jkao1) [reply app/models/article.rb]: That shouldn't happen. [sanitize](http://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html) is built for stripping HTML tags.

Comment (NicholasLYang) [reply app/models/article.rb]: Nah but you're running the split by space thingy before you run the html tag sanitize

Comment (NicholasLYang) [reply app/models/article.rb]: Also, I'm pretty sure `"hello world".split` works like `"hello world".split(" ")`

**Pull request merged**